tests/library/chromium/launcher.spec.ts

```diff
@@ -52,8 +52,8 @@ it('should open devtools when "devtools: true" option is given', async ({ browse
   await browser.close();
 });
 
-it('should return background pages', async ({ browserType, createUserDataDir, asset, channel }) => {
-  it.skip(channel === 'chromium-headless-shell', 'Headless Shell has no support for extensions');
+it('should return background pages', async ({ browserType, createUserDataDir, asset, headless }) => {
+  it.skip(headless, 'Headless has no support for extensions');
```

Comment:
My understanding is that `chromium-next` has support for extensions even in headless mode. Should this be the condition instead?

```suggestion
  it.skip(headless && channel !== 'chromium-next', 'chromium-headless-shell has no support for extensions');
```

Reply:
looks like it got resolved in the meantime :D